examples/inspection/plot_partial_dependence.py

```diff
@@ -138,78 +154,110 @@
 # also significantly cheaper to tune their hyperparameters (the default tend to
 # work well while this is not often the case for neural networks).
 #
-# Finally, as we will see next, computing partial dependence plots tree-based
-# models is also orders of magnitude faster making it cheap to compute partial
-# dependence plots for pairs of interacting features:
+# We will plot the partial dependence, both individual (ICE) and averaged one
```

Comment:
On the above paragraph, I think "default" should be "defaults" or "default parameters"